data/eos-autoupdater.conf

```diff
@@ -10,4 +10,5 @@
 #  3 - apply updates
 LastAutomaticStep=3
 IntervalDays=14
+RandomizedDelayDays=7
```

Comment:
I can’t think of a situation where you’d want one and explicitly not want teh other.